js/repeater-list.js

```diff
@@ -594,6 +594,7 @@
 		if (sortable) {
 			$both.addClass('sortable');
 			$div.on('click.fu.repeaterList', function () {
+				if(self.isDisabled) return;
```

Comment:
this is technically correct, but as per our style guide we are supposed to write it as follows:
`if (self.isDisabled) { return; }`